bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/StringUtils.java

```diff
@@ -293,8 +294,9 @@ public static String getExpressionStringValue(Object value, BLink parent) {
      *
      * @param value The value on which the function is invoked
      * @return Ballerina value represented by Ballerina expression syntax
+     * @throws BError for any parsing error
      */
-    public static Object parseExpressionStringValue(String value, BLink parent) {
+    public static Object parseExpressionStringValue(String value, BLink parent) throws BError {
```

Comment:
BError is not a checked exception right?

Reply:
Fixed in 7f00924